app/[...slug]/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { notFound } from "next/navigation"
+import React from "react"
+
+import BasicPageLayout from "@/components/pages/basicPageLayout/BasicPageLayout"
+import { fetcher } from "@/lib/graphql/fetchers/dpl-cms.fetcher"
+import {
+  GetPageByPathDocument,
+  GetPageByPathQuery,
+  NodeGoPage,
+} from "@/lib/graphql/generated/dpl-cms/graphql"
+
+async function page(props: { params: Promise<{ slug: string[] }> }) {
+  const params = await props.params
+
+  const { slug } = params
+
+  const data = await fetcher<GetPageByPathQuery, { path: string }>(GetPageByPathDocument, {
+    path: slug.join("/"),
+  })()
+
```

Comment:
Just spotted this. This is different than how we normally do it. And I think there is two problems here:
* We should use a common query client so we can control caching on a global scale if we want
* Consitency

We should do something like     return queryClient.fetchQuery({
      queryKey: useSearchWithPaginationQuery.getKey(args),
      queryFn: useSearchWithPaginationQuery.fetcher(args),
    })
.